packages/wonder-blocks-dropdown/components/select-item.js

```diff
@@ -66,75 +71,73 @@ type SelectProps = {|
 const StyledButton = addStyle("button");
 
 type CheckProps = {|
+    disabled: boolean,
     selected: boolean,
     pressed: boolean,
     hovered: boolean,
     focused: boolean,
 |};
 
-// TODO: return with Icon once it's been made!
 const Check = (props: CheckProps) => {
     const {selected, pressed, hovered, focused} = props;
-
     return (
-        <View style={[styles.checkContainer]}>
-            <svg
-                role="img"
-                width="16px"
-                height="16px"
-                viewBox="-1 0 12 12"
-                aria-hidden="true"
-            >
-                {selected && (
-                    <path
-                        fill={pressed || hovered || focused ? white : offBlack}
-                        d={checkIcon}
-                    />
-                )}
-            </svg>
+        <View style={[styles.check, !selected && styles.hide]}>
+            <Icon
+                icon={icons.check}
+                size={"small"}
+                color={pressed || hovered || focused ? white : offBlack}
+            />
         </View>
     );
 };
 
-// TODO: replace with Choice component once it's implemented
-const Checkbox = (props: CheckProps) => {
-    const {selected, pressed, hovered, focused} = props;
+// This is a smaller check specifically for use in checkboxes
```

Comment:
🛑 I'd expand this comment to explicitly warn devs to not copy it and to check with a designer first before using.